jobs/furnishings/flags.json

```diff
@@ -1,5 +1,6 @@
 {
     "flagValues": {
-        "enable-involuntary-dissolution": true
+        "enable-involuntary-dissolution": true,
+        "disable-dissolution-sftp-bcmail": true
```

Comment:
Since you're here, I think you can also add the FF for BCLaws 🙏

Reply:
added 😄